scalardl-node-client-sdk.js

```diff
@@ -39,7 +39,19 @@ function _createGrpcServices(properties) {
   const tlsEnabled = clientProperties.getTlsEnabled();
   let ledgerClient;
   let ledgerPrivilegedClient;
-  if (tlsEnabled) {
+  if (tlsEnabled && !ca) {
```

Comment:
IMHO, the following structure looks slightly better even though more nested.
```
if (tlsEnabled) {
    if (ca) {

    } else (

    }
} else {

}
```

Reply:
Sure, I will update it.